### DISCUSSION TITLE
New official release?

### DISCUSSION THREAD

**Discussion by adapowers** [fralau/mkdocs-macros-plugin#167] [Project life]
Hi there! Big fan of the plugin; I'm working to integrate it into my org's doc project, and I'm excited about all the functionality it'll unlock.

I noticed that the last GitHub release was v0.5.0, but the plugin is up to v1.0alpha in the tags.
Does this mean that soon we'll be getting a new 1.0 release?

My org requires version pinning for all pip installed MkDocs plugins in our Dockerfile, and I'd love to take advantage of all the stuff added since v0.5.0, but don't know if I should wait for a formal release or just try to pin to the 1.0alpha tag.

Thanks!

**Comment by fralau** [comment]
Hi, 

Glad that you like it! I am indeed planning to release the 1.0. 

BTW, did you try the alpha version? Did it work well for you?

**Reply by adapowers** [reply]
Thanks for the response! :) I haven't had a chance to try anything after 0.5.0—my org's admin prefers anything we add to our techdocs environment via `pip` to be version pinned, so I decided to wait until the next official release to upgrade.

**Reply by fralau** [reply]
Makes sense. The new version 1.0.1 is released!

**Reply by adapowers** [reply]
@fralau Congrats! I'm a bit confused though—the [releases page](https://github.com/fralau/mkdocs_macros_plugin/releases) still shows v0.5.0 as the latest. Will the project be using tags exclusively going forward? If so, that's fine, just need to figure out how to install a specific tag via pip. :)

**Reply by adapowers** [reply]
Actually, my bad—it seems like `pip3 install mkdocs_macros_plugin==1.0.1` works just as well as 0.5.0 did. I just don't understand pip, I guess. ;)
